rules/eslint/best-practices/off.js

```diff
@@ -1,139 +1,139 @@
 module.exports = {
     rules: {
         // Enforces getter/setter pairs in objects
-        'accessor-pairs': 0,
+        'accessor-pairs': 'off',
         // enforce return statements in callbacks of array’s methods
-        'array-callback-return': 0,
+        'array-callback-return': 'off',
         // treat var statements as if they were block scoped
-        'block-scoped-var': 0,
+        'block-scoped-var': 'off',
         // specify the maximum cyclomatic complexity allowed in a program
-        complexity: 0,
+        complexity: 'off',
         // require return statements to either always or never specify values
-        'consistent-return': 0,
+        'consistent-return': 'off',
         // specify curly brace conventions for all control statements
-        curly: 0,
+        curly: 'off',
         // require default case in switch statements
-        'default-case': 0,
+        'default-case': 'off',
         // enforces consistent newlines before or after dots
-        'dot-location': 0,
+        'dot-location': 'off',
         // encourages use of dot notation whenever possible
-        'dot-notation': 0,
+        'dot-notation': 'off',
         // require the use of === and !==
-        eqeqeq: 0,
+        eqeqeq: 'off',
         // make sure for-in loops have an if statement
-        'guard-for-in': 0,
+        'guard-for-in': 'off',
         //  Maximum classes per file
-        'max-classes-per-file': 0,
+        'max-classes-per-file': 'off',
```

Comment:
I feel that we should enforce one class per file.

Reply:
![giphub](https://media1.giphy.com/media/3otPoFUhO2Z9uG1Qvm/giphy.gif?cid=e1bb72ff5b63b88034786a2e451c8f24)

Reply:
😂👋

Reply:
![giphub](https://media0.giphy.com/media/B8ozpOCSLbAcM/giphy.gif?cid=e1bb72ff5b64763f3542785a63e11185)